lib/CMakeLists.txt

```diff
@@ -27,17 +27,66 @@ if(USE_SIFT_GPU)
     add_definitions(-DUSING_GPU)
 endif()
 
+############## LIBELAS ##############
+
+# use sse3 instruction set
+set (CMAKE_CXX_FLAGS "-msse3")
+
+if (UNIX)
+  find_package(OpenMP)
+endif()
+
+if(OPENMP_FOUND)
```

Comment:
Uhm, usually the `find_package(OpenMP)` directive should provide cmake variables like `OpenMP_*`, that are case-sensitive (e.g. see `OpenCV`). That might be working but I'll try the correct way as well. Be consistent also with the `message(STATUS OpenMP ...)` directives.